math/mean/mean.go

```diff
@@ -0,0 +1,27 @@
+package mean
+
+import (
+	"fmt"
+)
+
+func mean(values []float64) float64{
+
+	if len(values) <= 0 {
+		fmt.Println("No values detected")
```

Comment:
No need for this.

Reply:
Sorry to ask, is the meaning of "No need" referring to removing the whole validating condition of empty array ? Or just the line of code printing the message ?

One reason of not suggesting to remove the validating condition above is to consider the case where  empty array is passed into the function.

Reply:
Just the line printing the message.

Reply:
> Just the line printing the message.

Got it, just removed it in my latest committed work